Crash with unexpected command line parameter

An unexpected command line parameter, e.g. "test", leads to a crash:
```
./bazel-remote-2.3.8-linux-x86_64 --dir=/var/tmp/cache --max_size=10 test
```
```
2022/07/07 15:36:26 bazel-remote built with go1.18.2 from git commit 5909df148d2487a94e22abbd1b5b22f53c989652.
Error: bazel-remote does not take positional aguments
arg: test
[...]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0xb49da9]

goroutine 1 [running]:
main.run(0xc0003e2880)
	main.go:81 +0x15c9
github.com/urfave/cli/v2.(*App).RunContext(0xc00033eea0, {0xe218c8?, 0xc000034120}, {0xc000030080, 0x4, 0x4})
	external/com_github_urfave_cli_v2/app.go:322 +0x953
github.com/urfave/cli/v2.(*App).Run(...)
	external/com_github_urfave_cli_v2/app.go:224
main.main()
	main.go:58 +0x230
```

--- a/main.go
+++ b/main.go
@@ -77,8 +77,8 @@ func run(ctx *cli.Context) error {
 		}
 		fmt.Fprintf(ctx.App.Writer, "\n")
 
-		err = cli.ShowAppHelp(ctx)
-		return cli.Exit(err.Error(), 1)
+		_ = cli.ShowAppHelp(ctx)
+		os.Exit(1)
 	}
 
 	rlimit.Raise()

